### PR TITLE
Webex Channel Fix: personId -> roomId

### PR DESCRIPTION
**Pull request by isaac-philip**
This is a much needed fix because presently we are unable to have the rasa-bot respond back to the specific room that sends the message thus enabling `no reply` to the user

The fix is also presented at https://github.com/RasaHQ/rasa/pull/5460 but since it will be released at 1.11.0 thenn we need to hotfix it soon.

References RasaHQ/rasa#5460

**Proposed changes**:
- ...

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by isaac-philip** [comment]
@wochinge kindly help for this MR.

**Comment by sara-tagger** [comment]
Thanks for submitting a pull request 🚀 @erohmensing will take a look at it as soon as possible ✨

**Comment by isaac-philip** [comment]
Added the changelog, thanks.

File: changelog/6078.bugfix.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+bot will respond through `webex` channel to any user(room) communicating to it. Earlier bot responds only to fixed `roomId` set in credentials.yml config file correctly.
```
Comment (wochinge) [inline changelog/6078.bugfix.rst:1]:
```suggestion
The assistant will respond through the `webex` channel to any user (room) communicating to it. Before the bot responded only to a fixed ``roomId`` set in the ``credentials.yml``config file.
```

File: changelog/6078.bugfix.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+The assistant will respond through the `webex` channel to any user (room) communicating to it. Before the bot responded only to a fixed ``roomId`` set in the ``credentials.yml``config file.
```
Comment (wochinge) [inline changelog/6078.bugfix.rst:1]:
```suggestion
The assistant will respond through the `webex` channel to any user (room) communicating to it. Before the bot responded only to a fixed ``roomId`` set in the ``credentials.yml`` config file.
```

**Comment by isaac-philip** [comment]
Thanks for the help @wochinge ! 🥇

**Comment by isaac-philip** [comment]
@wochinge do you have any document you can share on the release strategy employed? I would like to learn from it.

**Comment by wochinge** [comment]
@isaac-philip Thanks for your question! We talked about your question internally, and we fully agree with you that the release process should be documented publicly and transparently. We will keep track of our progress towards this in https://github.com/RasaHQ/rasa/issues/6260 . 

Maybe this short info can help you out until #6260 is done:

1. We release Rasa Open Source `minors` in fixed 4 week cycles (currently on hold as we are working towards a Rasa Open Source 2.0 `major`)
2. `Micro` (also referred to as `patch`) releases are cut as required. They can only include bugfixes or documentation improvements, but no new features (following the [semantic versioning guidelines](https://semver.org/))

**Pull request merged**